impl/src/main/java/org/ehcache/loaderwriter/writebehind/WriteBehind.java

```diff
@@ -42,7 +42,9 @@
   
   /**
    * Gets the best estimate for items in the queue still awaiting processing.
-   * Not including elements currently processed
+   * Not including elements currently processed.
+   * Since the value returned is a rough estimate, it can sometimes be more than 
```

Comment:
So that's interesting - I was expecting you to put the javadoc on the specific implementation. But I realise this level also contains some wording that make the batching impl wrong.
So I suggest:
* Move the paragraph about the size being bigger down to the batching impl.
* Remove from here the bit about not including elements currently processed.